vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/pool/SqlConnectionPool.java

```diff
@@ -309,7 +313,7 @@ public class PooledConnection implements Connection, Connection.Holder {
       this.factory = factory;
       this.conn = conn;
       this.listener = listener;
-      this.lifetimeEvictionTimestamp = maxLifetime > 0 ? System.currentTimeMillis() + maxLifetime : Long.MAX_VALUE;
+      this.lifetimeEvictionTimestamp = maxLifetime > 0 ? System.currentTimeMillis() + maxLifetime + Math.max(0, ThreadLocalRandom.current().nextLong(0, jitter + 1)) : Long.MAX_VALUE;
```

Comment:
@priyanshu-d11 Do you think this value should be subtracted?
Since, the total eviction time should never exceed maxLifeTime (users expectation)
If yes, need to update corresponding test cases.